dockers/docker-base-bookworm/Dockerfile.j2

```diff
@@ -47,8 +47,8 @@ RUN apt update &&            \
         python-is-python3    \
         vim-tiny             \
         rsyslog              \
-# Install redis-tools
-        redis-tools          \
+# Install valkey-tools
+        valkey-tools         \
```

Comment:
Is it possible to enable both, and use a runtime config to pick one. No need to warm switch between them, but a /etc/sonic/config_db.json could change the config by a `config reload`.

Reply:
Will improve the PR, there may be some script will check the process name, so need figure out how dynamically create symbol link during config reload.